src/components/common/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+interface PropsTypes {
+  text: string;
+  fn: () => void;
```

Comment:
Rozważ zmianę nazwy tego propsa na inną np 'onclick'. Nazwa fn może być mało intuicyjna.
BTW czepiam się może nieco na siłę, ale po to jest code review. Wybacz 😂